modules/core/src/constants/profile-constants.ts

```diff
@@ -33,6 +33,7 @@ export class ProfileConstants {
     // SCIM2 schema IDs
     public static readonly SCIM2_CORE_USER_SCHEMA: string = "urn:ietf:params:scim:schemas:core:2.0:User";
     public static readonly SCIM2_ENT_USER_SCHEMA: string = "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User";
+    public static readonly SCIM2_WSO2_USER_SCHEMA: string = "urn:scim:custom:schema:User";
```

Comment:
shouldn't we use "urn:scim:wso2:schema:User" as dialect URI

Reply:
Fixed in commit 7b62e35d1d4208333be62a08306b80d17c3336da